skellytracker/process_folder_of_videos.py

```diff
@@ -122,9 +139,16 @@ def process_single_video(
     :param annotated_video_path: Path to save annotated video to.
     :return: Array of tracking data
     """
-    video_name = (
-        video_path.stem + "_mediapipe.mp4"
-    )  # TODO: fix it so blender output doesn't require mediapipe addendum here
+
+    if tracker_name == "OpenPoseTracker":
+        video_name = (
+            video_path.stem + "_openpose.avi"
```

Comment:
this is good for now but does feel hacky. If we run into more trackers with special file suffixes maybe we add this to the model_info